CMakeLists.txt

```diff
@@ -116,8 +123,22 @@ if(SLEIGHHOME_DEFAULT)
 	endif()
 endif()
 
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+	set(CMAKE_INSTALL_LIBDIR lib64 CACHE PATH "Output directory for libraries.")
+endif()
+
+macro(install_symlink filepath sympath)
+	install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink ${filepath} ${sympath})")
+	install(CODE "message(\"-- Created symlink: ${sympath} -> ${filepath}\")")
+endmacro(install_symlink)
+
 install(TARGETS core_ghidra DESTINATION "${RIZIN_INSTALL_PLUGDIR}")
+install_symlink(
+	"${CMAKE_INSTALL_PREFIX}/${RIZIN_INSTALL_PLUGDIR}/core_ghidra${CMAKE_SHARED_LIBRARY_SUFFIX}"
+	"${CMAKE_INSTALL_PREFIX}/${RIZIN_INSTALL_PLUGDIR}/libcore_ghidra${CMAKE_SHARED_LIBRARY_SUFFIX}")
```

Comment:
This is not a good idea, it will make rizin attempt to load the plugin twice. Why do you need the library with the `lib` prefix?

Reply:
@thestr4ng3r for this
```go
// #cgo pkg-config: rz_core rz_analysis rz_ghidra
```
and this is pkg-config:
```
Libs: -L"${libdir}" -L"${prefix}/@RIZIN_INSTALL_PLUGDIR@" -lcore_ghidra
Libs.private: -L"${libdir}" -L"${prefix}/@RIZIN_INSTALL_PLUGDIR@" -lcore_ghidra
```

Reply:
Can you use an absolute path to the .so file instead of `-L<path>`+`-lcore_ghidra`? Then the symlink should not be necessary anymore. rz-ghidra doesn't follow any typical lib filename conventions anyway, so I think this would make sense.

Reply:
Dropped the symlink and using absolute path now.